frontend/src/components/AccountAdventurersLog.tsx

```diff
@@ -0,0 +1,44 @@
+import { Account } from "../types/Account";
+import "../styles/AccountStats.css";
+import icons from "../assets";
+import "../styles/AccountAdventurersLog.css";
+
+type Props = {
+    accountData: Account;
+};
+
+const AccountAdventurersLog = ({ accountData }: Props) => {
+    const activities = accountData.activities;
+    console.log(activities);
+
+    return (
+        <div className="account-activities">
+            <div className="account-activities__title">Adventurer's Log</div>
+            <div className="account-activities__list">
+                {activities.map((activity) => {
+                    return (
+                        <div className="account-activities__stat">

```

Comment:
```suggestion
                        <div key={activity.date} className="account-activities__stat">
```
need to add a `key` property, can be anything unique